modules/misc/fontconfig.nix

```
@@ -32,10 +32,180 @@ in {
           <command>nix-env</command>.
         '';
       };
+      includes = mkOption {
+        type = types.listOf (types.submodule {
+          options = {
+            ignoreMissing = mkOption {
+              type = types.bool;
+              default = false;
+              description =
+                "Ignore this import if the path does not exist. Maps to the `ignore_missing` attribute.";
+            };
+            path = mkOption {
+              type = types.path;
+              description = "Path to be included.";
+            };
+          };
+        });
+        default = [ ];
+        description = "Paths to include.";
+      };
+      cacheDir = mkOption {
+        internal = true;
+        type = types.path;
+        description = "fontconfig cache directory";
+      };
+      dirs = mkOption {
+        type = types.listOf types.path;
+        default = [ ];
+        description = "Directories to load fonts from.";
+      };
+      aliases = mkOption {
+        type = types.listOf (types.submodule {
+          options = {
+            families = mkOption {
+              type = types.listOf types.str;
+              description = "Families to be matched.";
+            };
+            prefer = mkOption {
+              type = types.nullOr (types.listOf types.str);
+              default = null;
+              description =
+                "Families to be prepended before the matched family.";
+            };
+            accept = mkOption {
+              type = types.nullOr (types.listOf types.str);
+              default = null;
+              description = "Families to be placed after the matched family.";
+            };
+            default = mkOption {
+              type = types.nullOr (types.listOf types.str);
+              default = null;
+              description = "Families to be appended to the family list.";
+            };
+          };
+        });
+        default = [ ];
+        description = "Substitutions of one font family for another.";
+      };
+      matches = mkOption {
+        type = types.listOf (types.submodule {
+          options = {
+            tests = mkOption {
+              type = types.listOf (types.submodule {
+                options = {
+                  name = mkOption {
+                    type = types.str;
+                    description = "Attribute to match on.";
+                  };
+                  qual = mkOption {
```

I guess this stands for "qualifier"?

```suggestion
                  qualifier = mkOption {
```